docs/operator-manual/ingress.md

```diff
@@ -339,7 +339,7 @@ spec:
         - name: argocd-server
           port: 80
     - kind: Rule
-      match: Host(`argocd.example.com`) && Headers(`Content-Type`, `application/grpc`)
+      match: Host(`argocd.example.com`) && Header(`Content-Type`, `application/grpc`)
```

Comment:
Looking at the [Traefik Rules](https://doc.traefik.io/traefik/routing/routers/#rule), `Headers` seems to actually be correct